thirdparty/codemirror.css

```diff
@@ -0,0 +1,347 @@
+/* BASICS */
```

Comment:
The vendor doesn't provide one, but I can easily compress it, just hold on line please...